src/display/api.js

```diff
@@ -1534,8 +1534,21 @@ const PDFWorker = (function PDFWorkerClosure() {
         return worker.WorkerMessageHandler;
       }
       if ((typeof PDFJSDev !== 'undefined' && PDFJSDev.test('GENERIC')) &&
+          // eslint-disable-next-line no-undef
           (isNodeJS && typeof __non_webpack_require__ === 'function')) {
-        const worker = __non_webpack_require__(getWorkerSrc());
```

Comment:
Now that we don't use `__non_webpack_require__` in this `if` body anymore, I think we can also remove it from the `if` statement, together with its ESLint ignore line. If I'm not mistaken the same applies a few lines above.

Reply:
Considering this I also don't think it can hurt to do an extra check here. It's also not performance-heavy code.